mitmproxy/tools/console/master.py

```diff
@@ -123,12 +123,28 @@ def get_editor(self) -> str:
         else:
             return "vi"
 
+    def get_hex_editor(self) -> str:
+        if m := os.environ.get("MITMPROXY_HEXEDITOR"):
+            return m
+        for editor in "ghex", "hexedit", "bless", "wxhexeditor", "nano":
+            if shutil.which(editor):
+                return editor
+        if os.name == "nt":
+            for editor in "HxD", "WinVi", "Cygnus", "Frhed", "notepad":
+                if shutil.which(editor):
+                    return editor
+        else:
+            return "vi"
```

Comment:
`bless` command is completely different tool on macOS (in macOS, it is a tool to modify startup disk configuration / set volumes bootability), so it might be good to check Darwin or not.

also I think nano is not suitable for binary editor.

```suggestion
        editors = ["ghex", "hexedit", "wxhexeditor"]
        if platform.system() == "Darwin":
            editors = ["hexf"] + editors
        else:
            editors += ["bless"]
        if os.name == "nt":
            editors += ["HxD", "WinVi", "Cygnus", "Frhed", "notepad"]
        for editor in editors:
            if shutil.which(editor):
                return editor
        return "vi"
```

(note: `hexf` command is shortcut to Hex Fiend.app https://github.com/HexFiend/HexFiend )